resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -105,10 +105,27 @@ params=("--user" "<%= EZBake::Config[:user] -%>" "--group" "<%= EZBake::Config[:
 if [ -n "$os_version" ]; then params+=("--os-version" "$os_version"); fi
 if [ -n "$os_dist" ]; then params+=("--dist" "$os_dist"); fi
 
+params+=('--description' 'Puppet Labs <%= EZBake::Config[:project] %>. Contains: <%= Pkg::Config.config[:description] -%>')
+
 <% unless EZBake::Config[:terminus_info].empty? -%>
 DESTDIR="$basepath/termini" bash install.sh termini
 params+=('--build-termini')
 params+=('--termini-chdir' "$basepath/termini")
+termini_description='Termini for <%= EZBake::Config[:project] %>. Contains terminus for:'
+
+<% comma_counter = 0 -%>
+<% EZBake::Config[:terminus_info].each do |proj_name, info| -%>
+<% if comma_counter == 0
+     comma = ', '
+   else
+     comma = ''
+   end
+   comma_counter += 1
+%>
+termini_description+='<%= comma -%><%= proj_name -%> (version <%= info[:version] -%>)'
+<% end -%>
```

Comment:
The old packaging used newlines in its' descriptions, but I was having a hard time getting that working, so I added comma logic. https://github.com/jordansissel/fpm/issues/1468

Reply:
1. How does erb feel about indentation? This took me a minute to parse out, so I feel like it could be helpful for human eyes if the `if` block and `termini_description+=` were indented from the loop.

2. I assume you tested and saw this work how you wanted, but I'm having trouble seeing how it works. Should maybe the values be switched? Like you wouldn't want a comma after the `for: ` i.e. when comma_counter is 0, but you would want it for the rest? And then you'd probably not want it for the last one again, so maybe a case where there's no comma if `comma_counter == EZBake::Config[:terminus_info].count`?

Reply:
this is still being actively worked on, so yes, those were backwards.

I can probably get the if block better intented, but the termini_descriptor line is actually ending up in the shell script, so indenting it isn't going to work

Reply:
ooh, sorry, didn't know you were still working on all this, I'll hold off future review til you're ready. 😊